packages/std/src/errors/verification_error.rs

```diff
@@ -74,15 +100,33 @@ impl From<CryptoError> for VerificationError {
             CryptoError::PublicKeyErr { .. } => VerificationError::PublicKeyErr,
             CryptoError::GenericErr { .. } => VerificationError::GenericErr,
             CryptoError::InvalidRecoveryParam { .. } => VerificationError::InvalidRecoveryParam,
+            CryptoError::BatchErr { .. } => VerificationError::BatchErr,
         }
     }
 }
 
+impl From<StdError> for VerificationError {
+    fn from(original: StdError) -> Self {
+        VerificationError::standard_err(format!("{:?}", original))
+    }
+}
```

Comment:
Why is this conversion needed? This is getting circular now.

Reply:
Good catch. This was needed for when we used the safe `to_u32` converter. I'll remove this.